prosody/rootfs/etc/cont-init.d/10-config

```diff
@@ -31,6 +31,7 @@ fi
 
 if [[ ! -f $PROSODY_CFG ]]; then
     cp -r /defaults/* /config
+    tpl /defaults/prosody.cfg.lua > $PROSODY_CFG
```

Comment:
at this point you can remove the `cp -r` since we are templating all files. a `mkdir` would be necessary though.

Reply:
I'd argue the cp -r is more generally useful as it can continue to handle new files without any further changes.  I agree it's not doing much currently though, so I'm open to changing it if it's agreed that's better.